app/assets/javascripts/rad_common/autocomplete-rails.js

```diff
@@ -45,17 +45,18 @@
   {
       jQuery(".ui-autocomplete-input").each( function()
                                              {
-                                              display_autocomplete_error(this);
+                                              display_autocomplete_error(this, "id-element");
+                                              display_autocomplete_error(this, "association-element");
```

Comment:
Display errors for either the associated element or the id element depending on where there error was added in the model